build.gradle.kts

```diff
@@ -160,6 +161,34 @@ dependencies {
   testImplementation(libs.testng)
 }
 
+apply(plugin = "org.openapi.generator")
+openApiGenerate {
+  inputSpec = "$rootDir/docs/open-api/openapi.yaml".toString()
+  generatorName = "python"
+  outputDir = "$rootDir/clients/client-python"
+  modelPackage = "dto"
+  modelNameSuffix = "dto"
+  generateApiDocumentation.set(false)
+  generateModelDocumentation.set(false)
+  generateApiTests.set(false)
+  generateModelTests.set(false)
+  packageName.set("com.datastrato.gravitino.client")
+  cleanupOutput.set(true)
+
+  globalProperties.set(
+    mapOf(
+      "apis" to "false",
+      "models" to "Audit,Catalog,Metalake,Version"
```

Comment:
If we remove this, it will generate all DTO files in one package. Another solution, we can add a multi-task to generate different DTO on different tasks to separate them into different packages.